### DISCUSSION TITLE
Suggestion: show only minor & major releases in the "For you" feed

### DISCUSSION THREAD

**Discussion by alex-kim-dev** [community/community#14870] [Other Feature Feedback, Questions, & Ideas] [answered]
I'm not really interested in patch releases, but sometimes I like to keep up with new features & breaking changes.

**Comment by Tr1985** [comment]
https://github.com/github/feedback/discussions/14870#discussion-4007580

> Tôi không thực sự quan tâm đến các bản vá lỗi, nhưng đôi khi tôi muốn cập nhật các tính năng mới và những thay đổi đột phá.

**Comment by jfullstackdev** [comment, marked as answer]
hello guys, there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
